solutions/src/test/java/ru/sberbank/school/task10/FixedThreadPoolTest.java

```diff
@@ -0,0 +1,72 @@
+package ru.sberbank.school.task10;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class FixedThreadPoolTest {
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new FixedThreadPool(5);
+        threadPool.start();
+    }
+
+    @After
+    public void shutDown(){
```

Comment:
WhitespaceAround: &#39;{&#39; is not preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a773794163>*